bin/nodeserver.sh

```diff
@@ -3,4 +3,4 @@
 
 SYSTEM_TEST=$HOME/git/system-test
 export RUBYLIB=$SYSTEM_TEST/lib:$SYSTEM_TEST/tests
-exec env VESPA_HOME=$HOME/vespa VESPA_USER=$(id -un) VESPA_SYSTEM_TEST_USE_TLS=true RUBYLIB=$RUBYLIB ruby $SYSTEM_TEST/lib/node_server.rb "$@"
+exec env VESPA_HOME=$HOME/git/vespa VESPA_USER=$(id -un) VESPA_SYSTEM_TEST_USE_TLS=true RUBYLIB=$RUBYLIB ruby $SYSTEM_TEST/lib/node_server.rb "$@"
```

Comment:
Isn't `VESPA_HOME` expected to point to the Vespa install directory, which by default is `~/vespa` when installing locally?

Reply:
@vekterli  my bad, actually vespa c++ part is installed  into `$HOME/vespa` 🙈 
will revert that commit